boards/arm/mimxrt1020_evk/mimxrt1020_evk.dts

```diff
@@ -58,9 +58,11 @@
 };
 
 &flexspi0 {
+	reg = <0x402a8000 0x4000>, <0x60000000 0x800000>;
 	qspi0: qspi@60000000 {
-		/* ISSI IS25LP064A-JBLE */
-		reg = <0x60000000 0x800000>;
+		/* ISSI IS25WP064AJBLE */
+		compatible = "issi,is25wp064", "jedec,spi-nor";
```

Comment:
We don't have a `jedec,spi-nor` compatible yet; it's spelled `spi,flash` (supported by `drivers/flash/spi_nor`).  However I agree it should be `jedec,spi-nor` and in fact have support pending to be part of extending #11977 to include the `spi_nor` driver.

Do you want to leave it like this for now, or use `spi,flash` and I'll fix it later?

Reply:
Leave it for now, there isn't proper support for the 'issi' flash so just wanted the fix to the structure of the nodes.